app/controllers/report_controller/reports/editor.rb

```diff
@@ -1233,20 +1246,23 @@ def set_record_vars(rpt)
     rpt.sortby = @edit[:new][:sortby1] == NOTHING_STRING ? nil : []  # Clear sortby if sortby1 not present, else set up array
 
     # Add in the chargeback static fields
-    if rpt.db == "Chargeback" # For chargeback, add in static fields
-      rpt.cols = ["start_date", "display_range", "vm_name"]
+    if rpt.db.starts_with? "Chargeback" # For chargeback, add in static fields
+      rpt.cols = %w(start_date display_range)
+      name_col = "vm_name" if rpt.db == "ChargebackVm"
```

Comment:
this can be changed to name_col = rpt.db == "ChargebackVm" ? "vm_name" : "project_name"